hrjobcontract/js/test/controllers/form/form-pension_test.js

```diff
@@ -0,0 +1,136 @@
+define([
+  'mocks/data/contact',
+  'job-contract/app'
+], function(ContactMock) {
+  'use strict';
+
+  describe('FormPensionCtrl', function() {
+    var ctrl, $controller, $rootScope, $scope, $httpBackend, $q, ContactService, response, params;
+
+    beforeEach(module('hrjc'));
+
+    beforeEach(function() {
+      inject(function(_$controller_, _$rootScope_, _$httpBackend_, _$q_, _ContactService_) {
+        $controller = _$controller_;
+        $rootScope = _$rootScope_;
+        $httpBackend = _$httpBackend_;
+        ContactService = _ContactService_;
+        $q = _$q_;
+        params = {
+          contact_type: 'Organization',
+          contact_sub_type: 'Life_Insurance_Provider'
+        };
+        contactServiceSpy();
+      });
+    });
+
+    beforeEach(function() {
+      $httpBackend.whenGET(/action=get&entity=HRJobContract/).respond({});
+      $httpBackend.whenGET(/views.*/).respond({});
+    });
+
+    describe('On initialization', function() {
+      describe('Case I: when $scope.entity.pension.pension_type is truthy', function() {
```

Comment:
Since `pension_type` is not a boolean value rather it is a numerical value. I would rather prefer to call `when pension type is truthy`